docs/develop/tools-and-features/gsoc.md

```diff
@@ -0,0 +1,312 @@
+---
+title: GSOC
+id: gsoc
+---
+
+## Introduction
+
+The Graffiti Single Owner Chunk (GSOC) feature enables a single Bee *service* node to receive messages from multiple Bee *writer* nodes. It is based on a [Single Owner Chunk (SOC)](/docs/develop/tools-and-features/chunk-types/#single-owner-chunks), but unlike a standard SOC, which can be updated only by a single owner, GSOC allows multiple nodes to make updates. The GSOC address is derived so that it falls within the neighborhood of the service node, ensuring updates are automatically synced as part of the normal full node syncing process.  
+
+The service node determines the data used to derive the GSOC private key. Any node with access to this data can derive the same key and update the GSOC. The service node also defines a message format, filtering out non-conforming updates. Nodes which know both the GSOC key derivation data and the expected message format can send messages to the service node. Since only full nodes sync neighborhood chunks, the service node *must be a full node to receive GSOC updates*.  
```

Comment:
Defining the message format is not a hard requirement. You can subscribe (ws) to GSOC and receive every message. It is a requirement in `@anythread/gsoc`, but not in Bee API.